modules/ank/arrow-ank/src/main/kotlin/arrow/ank/main.kt

```diff
@@ -3,27 +3,25 @@
 package arrow.ank
 
 import arrow.effects.IO
+import arrow.effects.extensions.io.concurrent.concurrent
 import arrow.effects.extensions.io.fx.fx
 import arrow.effects.extensions.io.unsafeRun.runBlocking
 import arrow.unsafe
 import java.nio.file.Paths
 
-fun main(vararg args: String) {
-  unsafe {
-    runBlocking {
-      when {
-        args.size > 1 -> {
-          IO.fx().run {
-            ank(
-              Paths.get(args[0]),
-              Paths.get(args[1]),
-              args.drop(2),
-              interpreter
-            )
-          }
-        }
-        else -> throw IllegalArgumentException("Required first 2 args as directory paths in this order: <required: source> <required: destination> <optional: classpath entries, one per arg..>")
+fun main(vararg args: String) = unsafe {
```

Comment:
IIRC Kotlin now supports `suspend main`.

Reply:
`Error: Could not find or load main class MainKt` on Kotlin 1.3.31

Reply:
Maybe not?